Workbooks/Azure Monitor - Workspaces/Container Insights Usage/Container Insights Usage.workbook

```diff
@@ -0,0 +1,257 @@
+        "visualization": "linechart",
+        "tileSettings": {
+          "showBorder": false,
+          "titleContent": {
+            "columnMatch": "Namespace",
+            "formatter": 1
+          },
+          "leftContent": {
+            "columnMatch": "Total",
+            "formatter": 12,
+            "formatOptions": {
+              "palette": "auto"
+            },
+            "numberFormat": {
+              "unit": 17,
+              "options": {
+                "maximumSignificantDigits": 3,
+                "maximumFractionDigits": 2
+              }
+            }
+          }
+        },
+        "graphSettings": {
+          "type": 0,
+          "topContent": {
+            "columnMatch": "Namespace",
+            "formatter": 1
+          },
+          "centerContent": {
+            "columnMatch": "Total",
+            "formatter": 1,
+            "numberFormat": {
+              "unit": 17,
+              "options": {
+                "maximumSignificantDigits": 3,
+                "maximumFractionDigits": 2
+              }
+            }
+          }
+        }
+      },
+      "customWidth": "50",
+      "name": "query - 3",
+      "styleSettings": {
+        "maxWidth": "50",
+        "showBorder": true
+      }
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "ContainerLog\r\n|join(KubePodInventory)\r\non ContainerID\r\n|where TimeGenerated {Time_range}\r\n| summarize Total=sum(_BilledSize)/ 1000 by bin(TimeGenerated, 1d), LogEntrySource\r\n| render barchart\r\n",
+        "size": 0,
+        "showAnnotations": true,
+        "title": "Billable data per log-source type in MB",
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces"
+      },
+      "customWidth": "50",
+      "name": "query - 4",
+      "styleSettings": {
+        "maxWidth": "50",
+        "showBorder": true
+      }
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "ContainerLog\r\n|join(KubePodInventory) on ContainerID\r\n|where TimeGenerated {Time_range}\r\n| summarize Total= sum(_BilledSize)/ 1000 by bin(TimeGenerated, 1d), ClusterName\r\n| render linechart",
+        "size": 0,
+        "title": "Billable data per cluster name in MB",
+        "timeContext": {
+          "durationMs": 0
+        },
+        "timeContextFromParameter": "Time_range",
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces"
+      },
+      "customWidth": "50",
+      "name": "query - 6",
+      "styleSettings": {
+        "showBorder": true
+      }
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "AzureDiagnostics\r\n|where TimeGenerated {Time_range}\r\n| summarize Total= sum(_BilledSize)/ 1000 by bin(TimeGenerated, 1d), Category\r\n| render barchart\r\n",
```

Comment:
for one of the workspaces i have access to, there's no AzureDiagnostic table at all, so if this isn't guaranteed by the AKS solution, you might want to protect for this case?

![image](https://user-images.githubusercontent.com/10158007/75733809-18980380-5cab-11ea-8b60-bcaec47f8f66.png)


see [protect against a table that might not exist](https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Documentation/DataSources/LogsBestPracticesAndHints.md#protect-against-a-table-that-might-not-exist)

Reply:
Fixed with different approach